src/components/common/AddEditOrgPerson.vue

```diff
@@ -320,9 +253,6 @@ import { Getter } from 'vuex-class'
   ]
 })
 export default class AddEditOrgPerson extends Vue {
-  @Getter isBaseCompany!: boolean
-  @Getter getPeopleAndRolesResource!: PeopleAndRolesResourceIF
```

Comment:
Not needed here -- they are actually imported by the mixin. (All the code in the mixin is "added" to this component's class.)